app/src/main/java/de/westnordost/streetcomplete/util/ktx/LocalDate.kt

```diff
@@ -17,3 +36,59 @@ fun isApril1st(): Boolean {
     val now = LocalDate.now()
     return now.dayOfMonth == 1 && now.month == Month.APRIL
 }
+
+inline fun Instant_Companion.ofEpochMilli(epochMilliseconds: Long): Instant =
+    Instant.fromEpochMilliseconds(epochMilliseconds)
+
+inline fun LocalDate_Companion.now(): LocalDate = Clock.System.now().toLocalDate()
+
+inline fun LocalDate_Companion.of(year: Int, monthNumber: Int, dayOfMonth: Int): LocalDate =
+    LocalDate(year, monthNumber, dayOfMonth)
+
+fun LocalDateTime_Companion.now(): LocalDateTime =
+    Clock.System.now().toLocalDateTime(ZoneId.systemDefault())
+
+inline fun LocalDateTime_Companion.of(date: LocalDate, time: LocalTime): LocalDateTime =
+    LocalDateTime(date, time)
+
+inline fun LocalDateTime_Companion.ofInstant(instant: Instant, zone: ZoneId): LocalDateTime =
+    instant.atZone(zone)
+
+inline fun LocalTime_Companion.ofSecondOfDay(secondOfDay: Long): LocalTime =
+    LocalTime.fromSecondOfDay(secondOfDay.toInt())
+
+inline fun TimeZone_Companion.systemDefault(): ZoneId = ZoneId.currentSystemDefault()
+
+inline fun Instant.atZone(timeZone: ZoneId): LocalDateTime = this.toLocalDateTime(timeZone)
+
+inline fun Instant.toEpochMilli(): Long = this.toEpochMilliseconds()
+
+inline fun LocalDate.plusDays(days: Long): LocalDate = this.plus(days, DateTimeUnit.DAY)
+
+/** https://github.com/Kotlin/kotlinx-datetime#date--time-arithmetic */
+fun LocalDateTime.minusHours(hours: Long): LocalDateTime = ZoneId.systemDefault().let { tz ->
+    this.toInstant(tz).minus(hours, DateTimeUnit.HOUR, tz).toLocalDateTime(tz)
+}
+
+/** https://github.com/Kotlin/kotlinx-datetime#date--time-arithmetic */
+fun LocalDateTime.plusHours(hours: Long): LocalDateTime = ZoneId.systemDefault().let { tz ->
+    this.toInstant(tz).plus(hours, DateTimeUnit.HOUR, tz).toLocalDateTime(tz)
+}
```

Comment:
if `.plus(Long, DateTimeUnit)` is the kotlinx-datetime api to add/subtract hours/days/minutes/... from a datetime, it would probably be more consistent to have extension functions with the same signature on `LocalDateTime` too instead of trying to mirror the java.time API. It is not really more verbose than the java.time counterpart.

Reply:
https://github.com/streetcomplete/StreetComplete/pull/4471/commits/3a21417f2144b30f3943bde58def2ae5faee72c6